common/static/common/js/discussion/main.js

```diff
@@ -2,10 +2,9 @@
           DiscussionUser, DiscussionUserProfileView, DiscussionUtil */
 (function() {
     'use strict';
-    var DiscussionApp, DiscussionProfileApp;
 
     if (typeof Backbone !== "undefined" && Backbone !== null) {
-        DiscussionApp = {
+        window.DiscussionApp = {
```

Comment:
@bjacobel to give visibility that they are on `window` anyway. Note there are no var declaration for `DiscussionApp`, so they are essentially declared in global namespace. With `window` in front this smell is more explicit. 

And I'm afraid fixing it at this point does not make much sense - proper way to do that would be to use RequireJS, and since it is used in DiscussionXBlock we can't do that yet.